Source/WebCore/platform/ios/PlaybackSessionInterfaceLMK.mm

```diff
@@ -0,0 +1,153 @@
+{
+    return adoptRef(*new PlaybackSessionInterfaceLMK(model));
+}
+
+PlaybackSessionInterfaceLMK::PlaybackSessionInterfaceLMK(PlaybackSessionModel& model)
+    : PlaybackSessionInterfaceIOS(model)
+{
+    model.addClient(*this);
+
+    durationChanged(model.duration());
+    currentTimeChanged(model.currentTime(), [[NSProcessInfo processInfo] systemUptime]);
+    bufferedTimeChanged(model.bufferedTime());
+    OptionSet<PlaybackSessionModel::PlaybackState> playbackState;
+    if (model.isPlaying())
+        playbackState.add(PlaybackSessionModel::PlaybackState::Playing);
+    if (model.isStalled())
+        playbackState.add(PlaybackSessionModel::PlaybackState::Stalled);
+    rateChanged(playbackState, model.playbackRate(), model.defaultPlaybackRate());
+    seekableRangesChanged(model.seekableRanges(), model.seekableTimeRangesLastModifiedTime(), model.liveUpdateInterval());
+    canPlayFastReverseChanged(model.canPlayFastReverse());
+    audioMediaSelectionOptionsChanged(model.audioMediaSelectionOptions(), model.audioMediaSelectedIndex());
+    legibleMediaSelectionOptionsChanged(model.legibleMediaSelectionOptions(), model.legibleMediaSelectedIndex());
+    externalPlaybackChanged(model.externalPlaybackEnabled(), model.externalPlaybackTargetType(), model.externalPlaybackLocalizedDeviceName());
+    wirelessVideoPlaybackDisabledChanged(model.wirelessVideoPlaybackDisabled());
+}
+
+WebAVPlayerController *PlaybackSessionInterfaceLMK::playerController() const
+{
+    return nullptr;
+}
+
+PlaybackSessionInterfaceLMK::~PlaybackSessionInterfaceLMK()
+{
+    if (!m_playbackSessionModel)
+        return;
+
+    m_playbackSessionModel->removeClient(*this);
+    m_playbackSessionModel = nullptr;
+}
+
+void PlaybackSessionInterfaceLMK::durationChanged(double duration)
+{
+    UNUSED_PARAM(duration);
+}
+
+void PlaybackSessionInterfaceLMK::currentTimeChanged(double currentTime, double anchorTime)
+{
+    UNUSED_PARAM(currentTime);
+    UNUSED_PARAM(anchorTime);
+}
+
+void PlaybackSessionInterfaceLMK::bufferedTimeChanged(double bufferedTime)
+{
+    UNUSED_PARAM(bufferedTime);
+}
+
+void PlaybackSessionInterfaceLMK::rateChanged(OptionSet<PlaybackSessionModel::PlaybackState> playbackState, double playbackRate, double defaultPlaybackRate)
+{
+    UNUSED_PARAM(playbackState);
+    UNUSED_PARAM(playbackRate);
+    UNUSED_PARAM(defaultPlaybackRate);
+}
+
+void PlaybackSessionInterfaceLMK::seekableRangesChanged(const TimeRanges& timeRanges, double lastModifiedTime, double liveUpdateInterval)
+{
+    UNUSED_PARAM(timeRanges);
+    UNUSED_PARAM(lastModifiedTime);
+    UNUSED_PARAM(liveUpdateInterval);
+}
+
+void PlaybackSessionInterfaceLMK::canPlayFastReverseChanged(bool canPlayFastReverse)
+{
+    UNUSED_PARAM(canPlayFastReverse);
+}
+
+void PlaybackSessionInterfaceLMK::audioMediaSelectionOptionsChanged(const Vector<MediaSelectionOption>& options, uint64_t selectedIndex)
+{
```

Comment:
for this method and similar ones, can the parameter names just be omitted instead of using `UNUSED_PARAM`?